app/src/main/java/com/wafflestudio/snutt2/views/logged_out/TutorialPage.kt

```diff
@@ -53,6 +67,74 @@ fun TutorialPage() {
         }
     }
 
+    // accessToken을 SNUTT 서버에 보내 로그인을 실행한다.
+    val loginGoogleByAccessToken: (String) -> Unit = { googleAccessToken: String ->
+        coroutineScope.launch {
+            launchSuspendApi(
+                apiOnProgress = apiOnProgress,
+                apiOnError = apiOnError,
+                loadingIndicatorTitle = context.getString(R.string.sign_in_sign_in_button),
+            ) {
+                userViewModel.loginGoogle(
+                    googleAccessToken,
+                )
+                homeViewModel.refreshData()
+                navController.navigateAsOrigin(NavigationDestination.Home)
+            }
+        }
+    }
+
+    // 계정을 선택하면 authCode가 받아지는데, 그 authCode를 구글 API로 보내 accessToken을 얻는다.
+    val getAccessTokenByAuthCode: (String) -> Unit = { authCode: String ->
+        coroutineScope.launch {
+            val response = userViewModel.getAccessTokenByAuthCode(
+                authCode = authCode,
+                clientId = clientId,
+                clientSecret = clientSecret,
+            )
+            if (response.accessToken == null) {
+                context.toast(context.getString(R.string.sign_in_sign_in_google_failed_unknown))
+            } else {
+                loginGoogleByAccessToken(response.accessToken)
+            }
+        }
+    }
```

Comment:
```suggestion
    val handleGoogleSignInCallback: (String) -> Unit = { authCode: String ->
        coroutineScope.launch {
            launchSuspendApi(
                apiOnProgress = apiOnProgress,
                apiOnError = apiOnError,
                loadingIndicatorTitle = context.getString(R.string.sign_in_sign_in_button),
            ) {
                val googleAccessToken = userViewModel.getAccessTokenByAuthCode(
                    authCode = authCode,
                    clientId = clientId,
                    clientSecret = clientSecret,
                )
                if (googleAccessToken != null) {
                    userViewModel.loginGoogle(googleAccessToken)
                    homeViewModel.refreshData()
                    navController.navigateAsOrigin(NavigationDestination.Home)
                } else {
                    context.toast(context.getString(R.string.sign_in_sign_in_google_failed_unknown))
                }
            }
        }
    }
```
둘이 합치고 coroutineScope.launch도 중복으로 쓰는 거 없애고 이름도 바꾸면 좋을듯